bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BJSON.java

```diff
@@ -129,6 +129,21 @@ public BJSON(InputStream in, String schema) {
         }
     }
 
+    /**
+     * Create a {@link BJSON} from a {@link InputStream} with a given charset.
+     *
+     * @param in      InputStream of the json content
+     * @param schema  Schema of the json
+     * @param charset Charset value to be used in parsing
+     */
+    public BJSON(InputStream in, String schema, String charset) {
```

Comment:
This constructor seems unused and schema parameter is also not used. Do we really need this constructor?

Reply:
Yeah we need this. I have mistakenly used the other method. This is used now in charset cases.